fnet/src/vespa/fnet/transport_thread.h

```diff
@@ -43,10 +43,11 @@ class FNET_TransportThread : public FastOS_Runnable
     Selector                 _selector;       // I/O event generator
     FNET_PacketQueue_NoLock  _queue;          // outer event queue
     FNET_PacketQueue_NoLock  _myQueue;        // inner event queue
-    std::mutex               _lock;           // used for synchronization
-    std::condition_variable  _cond;           // used for synchronization
+    std::mutex               _qLock;          // protects the Q
+    std::mutex               _lock;           // used for synchronization during shutdown
```

Comment:
you should rename this lock instead of the other lock

Reply:
Fixed,